commerce-test/src/testIntegration/java/com/liferay/commerce/order/stock/test/OrderStockManagementTest.java

```diff
@@ -123,16 +148,23 @@ public void testProductAvailability() throws Exception {
 		);
 
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_company.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addB2CCommerceOrder(
-			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
+			_company.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId(),
+			_commerceChannel.getSiteGroupId());
+
+		_commerceOrders.add(commerceOrder);
 
-		CPInstance cpInstance = CPTestUtil.addCPInstance(_group.getGroupId());
+		CPInstance cpInstance = CPTestUtil.addCPInstance(_company.getGroupId());
 
 		CommerceInventoryWarehouse commerceInventoryWarehouse =
 			CommerceInventoryTestUtil.addCommerceInventoryWarehouse(
-				_group.getGroupId());
+				_company.getGroupId());
+
+		CommerceTestUtil.addCommerceChannelRel(
```

Comment:
why groupId? and if the relationship is just with warehouse the method name should be more specific